### ISSUE TITLE
How do we handle detailed policy?

### ISSUE THREAD

**Issue by Floppy** [openpolitics/manifesto#578]
More often now, people are refining outlined policies to add extra detail. This is good, and we don't want to stop it, but how do we align that with the need to be concise and readable?

We could:

* Put details in footnotes - could get long still, though.
* Put details in an appendix - but how do we link, and how do we make the process of setting that up simple?
* Separate policies into separate files, with a summary and detail section, then render into different places differently. Big change to the setup though, although we've discussed it before.

Any thoughts?

**Comment by areteh** [comment]
I agree that it is good - if anything we want to have a tiered systemt that allows multiple tiers eg highlights, specific policies and details/data. Ideally the detail level allows us to work towards a fully costed approach - otherwise it becomes too easy to simply suggest ideas that although popular would be unaffordable. 

This would probably rule the first two options out. To simplify the 3rd, could we link to a specific wiki for the given policy. We could then differentiate between policies that had been elaborated on and those that were headline ideas.

**Comment by Xyleneb** [comment]
In transport.md would be a basic policy like "We will widen the roads".
Then within the amendment I'd put in sub-script reference numbers. "We will widen the roads[1], [2]." 
Those reference numbers would either point to links to external sources/evidence, or to issues #457 and #546, etc.
By linking to Issues I'm able to flesh out the policy, without using my own blog or external publishing, which could be unreliable (and the whole "extended" policy could disappear one morning).

The only problems are that:
1) The distinction between external justification/evidence and your own writing wouldn't be clear. Both look the same on the page. You'd need a reference section (similar to the footnotes) that could resemble the way wikipedia does things.
2) Issue pages aren't treated as policy, so the "full" policies lack democratic oversight.
3) I also don't know if you have a storage limit. You could potentially be talking multiple 400-page documents as "Issues". If you do have a limit then you'd have to have something like votes to purge, or auto-expiry.

I don't know if that way is the best way. But the sooner you establish a method of doing it, the sooner I can include specific stuff relating to transport.
